### PR TITLE
Provide function for comma-separated string inputs

### PR DESCRIPTION
**Pull request by lucperkins**
Add helper function for comma-separated strings

### CODE REVIEW

File: src/inputs.ts [hunk 1]
```diff
@@ -11,6 +11,14 @@ const getBool = (name: string): boolean => {
   return actionsCore.getBooleanInput(name);
 };
 
+/**
+ * Convert a comma-separated string input into an array of strings, stripping all whitespace from the string.
```
Comment (cole-h) [inline src/inputs.ts:15]: IMHO the whitespace stripping should be up to the consumer.

**Review by cole-h — APPROVED** [review]
Otherwise, LGTM.

File: src/inputs.ts (hunk 1 above)
Comment (lucperkins) [reply src/inputs.ts:15]: Okay, I'll have separate functions

Comment (lucperkins) [reply src/inputs.ts:15]: Actually, I'll make it an optional function arg

**Pull request merged**